### ISSUE TITLE
Lines containing syntax errors are impossible to read

### ISSUE THREAD

**Issue by mojodna** [arcticicestudio/nord-jetbrains#60]
![image](https://user-images.githubusercontent.com/45/59316741-cfafa300-8c74-11e9-8f9c-cfd85ead65dd.png)

**Comment by arcticicestudio** [comment]
Hi @mojodna :wave:, thanks for your contribution :+1:
I remember I once tried to fix this by reducing the opacity of the color, but I think there were problems with IntelliJ's highlighting API regarding the rendering and support of transparent colors.
We can try again if this has been solved and maybe use the same colors like for the UI theme's [`FileColors`][ref] keys (`1A` = 26% opacity).

[ref]: https://github.com/arcticicestudio/nord-jetbrains/blob/aa7f39e8b751a70f499228cb0fe4c2e8a9a3befb/src/nord.theme.json#L126-L132

**Comment by arcticicestudio** [comment]
I guess the problem here is that the whole line is highlighted with `nord11` because of the styles defined for IntelliJ's _Code Inspection_ feature (`ERRORS_ATTRIBUTES` editor scheme key) mixed with the foreground color for syntax errors that is also `nord11`.
This results in `nord11` foreground on `nord11` background.

I've tried to use a opacity of 60% for `nord11` as background color instead, but the result is a "dirty redish" color due to the base editor background color, so I though about a better way of highlighting these errors.

<p align="center">Reduced opacity of 60%</p>
<p align="center"><img src="https://user-images.githubusercontent.com/7836623/59513773-d1b77300-8ebb-11e9-95d8-ddb86a988c82.png"></p>

I came up with a styling that doesn't make use of a background color at all based on the fact that it is a duplicate highlighting. If there's a syntax error the invalid tokens will be highlighted with `nord11` due to the syntax error so there is no need to additionally colorize the background with `nord11` resulting in unreadable text due to the same foreground color.

The new styles for _Code Inspection_ errors will simply use `nord11` as foreground color with a bold underline and the already used stripe next to the line numbers. This design decision also comes with a change for the “Unknown Symbol“ highlighting that will now use a dotted underline instead to differentiate from the new error styles.

<p align="center">Reduced opacity of 60%</p>
<p align="center"><img src="https://user-images.githubusercontent.com/7836623/59513841-f875a980-8ebb-11e9-953d-eb8a7eb9f8b6.png"></p>
